source/wp-content/themes/wporg-parent-2021/sass/blocks/_preformatted.scss

```diff
@@ -0,0 +1,5 @@
+.wp-block-preformatted {
+	white-space: pre;
```

Comment:
This is causing an issue:

[Video Shortcode](https://wordpress.org/documentation/article/video-shortcode/):

<img width="333" alt="Screenshot 2023-04-13 at 1 01 19 PM" src="https://user-images.githubusercontent.com/1657336/231648793-e3363cbe-735f-42b0-848d-667582a2c5ef.png">

Reply:
I've fixed the content there — looks like it was hit by [this shortcode issue](https://core.trac.wordpress.org/ticket/55996#comment:29), it was not meant to actually embed any videos. If a video is embedded not in a `pre`, it should wrap the error message just fine.